src/components/Modals/Send/AddressInput/AddressInput.tsx

```diff
@@ -34,7 +34,6 @@ export const AddressInput = ({ rules, placeholder, enableQr = false }: AddressIn
     }) => (
       <Input
         spellCheck={false}
-        autoFocus
```

Comment:
@reallybeard is this removal on purpose?

Reply:
Yah the autoFocus on mobile can cause issues with the keyboard popping up, or it just doesn't work. I guess we can flag it on or off based on isMobile.

The auto focus pattern isn't consistent across apps from what i've seen.

Reply:
Sounds good to me!